### PR TITLE
DOCS-282 - Add Cloud SIEM example for Copilot

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request adds a Cloud SIEM example section to the Copilot preview article: https://help.sumologic.com/docs/search/copilot

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

[DOCS-282](https://sumologic.atlassian.net/browse/DOCS-282)

### CODE REVIEW

File: docs/search/copilot.md [hunk 1]
```diff
@@ -136,28 +138,63 @@ If needed, you can edit your log search query code.
...
+1. Rowena knows what she is looking for. So, she asks:
+   ```
+   Count logs by action. Sort the results.
+   ```
+   <img src={useBaseUrl('img/search/copilot/copilot-cloud-siem-1.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="500" />
+1. As soon as she does that, she can look at the **Suggestions** section on the right. These suggestions are curated based on their relevance to this Cloud SIEM source. Rowena picks a suggestion to compare results to the last hour:
+   ```
+   Count logs by action. Sort the results. versus the previous 1h
+   ```
+   Notice the system translated this suggestion to a log query and rendered results as a bar graph with no user input. <br/><img src={useBaseUrl('img/search/copilot/copilot-cloud-siem-2.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="800" />
```
Comment (JV0812) [inline docs/search/copilot.md:184]:
```suggestion
   Notice the system translated suggestion to a log query and rendered results as a bar graph with no user input. <br/><img src={useBaseUrl('img/search/copilot/copilot-cloud-siem-2.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="800" />
```

**Review by JV0812 — APPROVED** [review]
Great work, @jpipkin1.

File: docs/search/copilot.md (hunk 1 above)
Comment (jpipkin1) [inline docs/search/copilot.md:184]:
```suggestion
   Notice the system translated the suggestion to a log query and rendered results as a bar graph with no user input. <br/><img src={useBaseUrl('img/search/copilot/copilot-cloud-siem-2.png')} alt="Copilot tab" style={{border: '1px solid gray'}} width="800" />
```

**Comment by jpipkin1** [comment]
Thanks for the review, @JV0812!

@kimsauce, could you review this PR, since you own the article?

**Review by kimsauce — CHANGES_REQUESTED** [review]
Hi @jpipkin1 ! In my previous examples, I rewrote them so that they don't mention a person's name and instead I use "you" or "we". What do you think?

**Comment by jpipkin1** [comment]
Great suggestion, @kimsauce. It's updated now.

**Review by kimsauce — APPROVED** [review]

**Pull request merged**